x-pack/plugins/runtime_fields/public/components/runtime_field_form/runtime_field_form.scss

```diff
@@ -0,0 +1,5 @@
+.runtimeFieldEditor_form {
```

Comment:
Our css class naming follows the BEM schema and is made unique with a three-letter prefix to avoid style conflicts across applications. In this case, I would propose using something like `.rtfEditor`

Reply:
Cool. Just so I understand you correctly, if I was going to correct this it would be `.rtfEditor__form` and not `.rtfEditor` right? As "form" is a child of the editor.

Reply:
There's some subjectivity, but yeah... in this case `rtfEditor` is the entity/block and form is the element.